documentation/domains/Cluster.json

```diff
@@ -645,6 +665,15 @@
         }
       }
     },
+    "Introspector": {
```

Comment:
This stanza doesn't look like it belongs at the cluster scope.

Reply:
There is a general issue where all domain level stanza are included at the cluster scope incuding 'AdminServer` and `domainHomeSourceType` etc. This needs to be investigated seprately.

Reply:
Fixed in [a6b2383](https://github.com/oracle/weblogic-kubernetes-operator/pull/3914/commits/a6b23837e674e333f027e573edba8c9dedd57ca5).